peachjam/admin.py

```diff
@@ -1153,6 +1153,39 @@ class ArticleAttachmentInline(BaseAttachmentFileInline):
     model = ArticleAttachment
     extra = 1
 
+    def attachment_url(self, obj):
+        return reverse(
```

Comment:
It strikes me that this is the `get_absolute_url` for this attachment object.

Reply:
right. I've added it as such and changed the links including in the template